### DISCUSSION TITLE
How to implementing infinitescroll prisma with Graphql pagination?

### DISCUSSION THREAD

**Discussion by rahXephonz** [prisma/prisma#11525] [Q&A] [answered]
Hello everyone 👋

Is there a method or something that i can read some tutorial, reference for implementing infinitescroll with prisma and graphql? Please tell me and if you have some example please drop it in here.

**Comment by ruheni** [comment, marked as answer]
Hi there 👋🏽

We don't have an example using infinite scroll with Prisma. However, you can refer to the [Fullstack Series – Part 2](https://www.prisma.io/blog/fullstack-nextjs-graphql-prisma-2-fwpc6ds155#pagination) covers pagination with GraphQL. The pagination section includes a `fetchMore` function. 

Infinite scrolling is an implementation detail on the frontend, and [Traversy Media has a tutorial](https://www.youtube.com/watch?v=gk_6BKiy6X4) on how to implement it on the frontend. I hope you find this helpful. 🙂

**Reply by rahXephonz** [reply]
Thanks! I will appreciate it sir, and time to find new stuf idea!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
